client/shaders/nodes_shader/opengl_fragment.glsl

```diff
@@ -199,7 +199,9 @@ void main(void)
 #endif
 
 	if (fogDistance != 0.0) {
-		float d = clamp((fogDistance - length(eyeVec)) / (fogDistance * 0.6), 0.0, 1.0);
+		// On some systems, the order of operations matters here.
+		// See the commit message of the commit that changed this.
+		float d = clamp(1.0 / 0.6 - length(eyeVec) / (fogDistance * 0.6), 0.0, 1.0);
```

Comment:
I would prefer to keep it fairly precise at 40% even if not visually noticeable.
How about 1.667?